src/clusterfuzz/_internal/metrics/logs.py

```diff
@@ -412,6 +389,8 @@ def cloud_label_filter(record):
 
   handler.addFilter(cloud_label_filter)
   handler.setLevel(logging.INFO)
+  formatter = JsonFormatter()
+  handler.setFormatter(formatter)
 
   logging.getLogger().addHandler(handler)
```

Comment:
will we still have file logs in the bots directory this way? This can be a followup TODO if not. 

File logs have been very useful in the past for debugging, when cloud logs somehow don't exist (e.g. network or some other issue).

Reply:
> will we still have file logs in the bots directory this way?

Yes, the file logic is configured [here](https://github.com/google/clusterfuzz/blob/3816d4176618d5c1e896b45d659be8935d222b9b/src/clusterfuzz/_internal/metrics/logs.py#L414-L415), and it's [set to run by default](https://github.com/google/clusterfuzz/blob/3816d4176618d5c1e896b45d659be8935d222b9b/src/clusterfuzz/_internal/metrics/logs.py#L82-L84). I'm not changing the logic there. I didn't deeply investigate how `logging.config` sets the file logging itself, but when I SSHed into the machines, I found the logs in the current folder.